e2e-tests/README.md

```diff
@@ -1,120 +1,54 @@
-Web QA Tests for crash-stats.mozilla.com
-===============================
-
-Mozilla's Crash Reporting System
--------------------------------------------
-
-Thank you for checking out Mozilla's Socorro test suite! Mozilla and the Web
-QA team are grateful for the help and hard work of many contributors like yourself.
-The following contributors have submitted pull requests to Socorro-Tests:
-
-https://github.com/mozilla/Socorro-Tests/contributors
+End-to-End Tests for Socorro
+=============================
 
 Continuous Integration
 ----------------------
-[![license](https://img.shields.io/badge/license-MPL%202.0-blue.svg)](https://github.com/mozilla/Socorro-Tests/tree/master#license)
 [![travis](https://img.shields.io/travis/mozilla/Socorro-Tests.svg?label=travis)](http://travis-ci.org/mozilla/Socorro-Tests/)
-[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.stage.svg?label=stage)](https://webqa-ci.mozilla.com/job/socorro.stage/)
-[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.prod.svg?label=prod)](https://webqa-ci.mozilla.com/job/socorro.prod/)
-[![requirements](https://img.shields.io/requires/github/mozilla/Socorro-Tests.svg)](https://requires.io/github/mozilla/Socorro-Tests/requirements/?branch=master)
-
-Getting involved as a contributor
----------------------------------
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.stage.saucelabs.svg?label=stage)](https://webqa-ci.mozilla.com/job/socorro.stage.saucelabs/)
+[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.prod.saucelabs.svg?label=prod)](https://webqa-ci.mozilla.com/job/socorro.prod.saucelabs/)
 
-We love working with contributors to improve the Selenium test coverage for
-Socorro-Tests but it does require a few skills.  You will need to be familiar
-with Python, Selenium, and have a working knowledge of GitHub.
 
-If you are comfortable with Python, it's worth having a look at the Selenium
-framework to understand the basic concepts of browser-based testing and the
-page objects pattern.
+This directory holds Socorro client-based end-to-end tests which is why they're different than the rest of the code in this repository.
 
-If you need to brush up on programming but are eager to start contributing
-immediately, please consider helping out by doing manual testing.  You can
-help find bugs in Mozilla [Firefox][firefox] or find bugs in the Mozilla web
-sites tested by the [Web QA][webqa] team.  We have many projects that would be
-thrilled to have your help!
-
-To brush up on Python skills before engaging with us, [Dive Into Python][dive]
-is an excellent resource.  MIT also has [lecture notes on Python][mit] available
-through their open courseware.  The programming concepts you will need to know
-include functions, working with classes, and the basics of object-oriented
-programming.
+* PyPOM
+* pytest
+* pytest-selenium
+* pytest-xdist
+* requests
```

Comment:
My preference would be to remove this list and add most make mention that there are specific python packages the tests use and suggest the reader review the `requirements.txt` file.